src/main/java/com/yahoo/bullet/bql/tree/SelectItemNode.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright 2018, Oath Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+
+/*
+ * Adapted and modified from the Presto project:
+ * https://github.com/prestodb/presto/blob/1898faf2ec4881709c9b8197e8332f302d618875/presto-parser/src/main/java/com/facebook/presto/sql/tree/SelectItem.java
+ */
+package com.yahoo.bullet.bql.tree;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import java.util.Objects;
+
+@Getter
+@RequiredArgsConstructor
+public class SelectItemNode extends Node {
+    private final boolean all;
+    private final ExpressionNode expression;
+    private final IdentifierNode alias;
+
+    @Override
+    public <R, C> R accept(ASTVisitor<R, C> visitor, C context) {
+        return visitor.visitSelectItem(this, context);
+    }
+/*
```

Comment:
Why is this commented out?

Reply:
Originally had equals/hashCode on all nodes, but I decided it was probably only necessary for ExpressionNodes. I'm a bit undecided because equals/hashCode could be useful here if I used a Set in ProcessedQuery, but for now, it's a List.

Reply:
I think we should just to be safe? It's only a couple where it's commented out.